experimental/single_number_formatter/Cargo.toml

```diff
@@ -0,0 +1,39 @@
+[package]
+name = "icu_singlenumberformatter"
```

Comment:
question: I don't really understand  the scope of this crate. Do we need separate crates for the "single" case? I figured we'd have a `number_formatter` and `units` crate, containing single and multi cases. Thoughts? cc@sffc @robertbastian ?

(Also the crate name should probably be `icu_single_number_formatter`)

Reply:
I have already filed an issue to discuss the naming of the crates or splitting them before moving forward in the next PRs.

Reply:
https://github.com/unicode-org/icu4x/issues/3676